woocommerce/woocommerce
# [Accessibility] Update the breadcrumb so the last item is an accessible link

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This was an issue fixed while exploring **My Account nav missing aria-current page indicator** 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure that the last item in the breadcrumb is a link
2. Inspect the code and make sure that the last link in the breadcrumb has an `aria-current` attribute set to `page`
3. Using a screen reader, make sure that when the focus is on the last link that it announces that it is the `current page`



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Turns the last item in the breadcrumb into a link with an `aria` attribute that announces to screen readers that the link is the current page.

</details>
<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
